tests/test_tifffile.py

```diff
@@ -133,6 +129,8 @@ def test_tifffile_reading_writing():
 
 def test_imagej_hyperstack(tmp_path):
     # create artifical hyperstack
+    import tifffile
+
```

Comment:
Do we need this here? We have the `importorskip` behavior at the top.

Reply:
@anjos Same as earlier in the review; wouldn't `importskip` at the top suffice?

Reply:
Yes, will fix.